static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -303,33 +303,40 @@ define([
     }
 
     function setupAdvertising() {
-        // if we don't already have googletag, create command queue and load it async
-        if (!window.googletag) {
-            window.googletag = { cmd: [] };
-            // load the library asynchronously
-            require(['js!googletag.js']);
-        }
+        return new Promise(function (resolve) {
+            // if we don't already have googletag, create command queue and load it async
+            if (!window.googletag) {
+                window.googletag = googletag = { cmd: [] };
+                // load the library asynchronously
+                require(['js!googletag.js']);
+            } else {
+                googletag = window.googletag;
+            }
 
-        if (prebidEnabled) {
-            prebidService = new PrebidService();
-        }
+            if (prebidEnabled) {
+                prebidService = new PrebidService();
+            }
 
-        window.googletag.cmd.push = raven.wrap({ deep: true }, window.googletag.cmd.push);
+            googletag.cmd.push = raven.wrap({ deep: true }, googletag.cmd.push);
 
-        window.googletag.cmd.push(function () {
-            renderStartTime = new Date().getTime();
+            googletag.cmd.push(
```

Comment:
I didn't have any problem during my tests, but I'll run some more just to make sure. Would be foolish of them to replace a native function and changing the signature though.